packages/runner/lib/exec.ts

```diff
@@ -122,7 +122,7 @@ export async function exec(
                 const valOutput = validateData({
                     version: nangoProps.syncConfig.version || '1',
                     input: output,
-                    modelName: nangoProps.syncConfig.models.length > 0 ? nangoProps.syncConfig.models[0] : undefined,
+                    modelName: nangoProps.syncConfig.models!.length > 0 ? nangoProps.syncConfig.models![0] : undefined,
```

Comment:
I get eslint non-null assertion errors here. 

```suggestion
                    modelName: nangoProps.syncConfig.models && nangoProps.syncConfig.models.length > 0 ? nangoProps.syncConfig.models[0] : undefined,
```

Reply:
Good point